File: Linguagem C Exercicios/Vetores/ABAIXO DA MEDIA.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;
    double soma,media;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d",&n);

    double vet[n];

    //Lendo o vetor
    for(int i=0;i<n;i++){
        printf("Digite um numero: ");
        scanf("%lf",&vet[i]);
    };

    printf("\n");

    //Somatoria e media dos numeros do vetor
    soma = 0;
    media = 0;
    for (int i=0;i<n;i++){
        soma = soma + vet[i];
    };
    media = soma / n;
    printf("MEDIA DOS VETORES = %.3lf\n",media);

    //Imprimir os elementos abaixo da média
    printf("ELEMENTOS ABAIXO DA MEDIA:\n");
    for (int i=0;i<n;i++){
        if (vet[i] < media){
            printf("%.1lf\n",vet[i]);
        };
    };

    return 0;
}

File: Linguagem Java/Exercicios/AbaixoDaMedia.java
import java.util.Locale;
import java.util.Scanner;

public class AbaixoDaMedia {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		double soma,media;
		
		System.out.print("Quantos elementos vai ter o vetor? ");
		n = sc.nextInt();
		
		double vet[] = new double[n];
		
		// Lendo o vetor
		for (int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();			
		};
		
		System.out.println();
		// Soma e media dos vetores
		soma = 0;
		for (int i=0;i<n;i++) {
			soma = soma + vet[i];			
		};
		media = soma / n;
		System.out.println("MEDIA DO VETOR = "+String.format("%.3f", media));
		
		// Elemento abaixo da media
		System.out.println("ELEMENTOS ABAIXO DA MEDIA: ");
		for (int i=0;i<n;i++) {
			if (vet[i] < media) {
				System.out.println(vet[i]);				
			};			
		};
		
		sc.close();
	}

}
